tools/pot/setup.py

```diff
@@ -65,21 +65,28 @@ def get_version():
     sys.argv.remove('--install-dev-extras')
 
 INSTALL_REQUIRES = [
-    "scipy>=1.5.4",
+    "scipy~=1.7; python_version == '3.7'",
+    "scipy>=1.8; python_version >= '3.8'",
     "jstyleson>=0.0.2",
     "numpy>=1.16.6,<=1.23.1",
     "addict>=2.4.0",
     "networkx~=2.5;python_version<='3.6'",
     "networkx<2.8.1;python_version>'3.6'",
     "tqdm>=4.54.1",
     "texttable>=1.6.3",
-    "pandas~=1.1.5",
+    "pandas~=1.3.5",
     "openvino-telemetry>=2022.1.0"
 ]
 
 ALGO_EXTRAS = []
 
-DEV_EXTRAS = ['pytest>=5.0,<=7.0.1', 'openpyxl==2.6.4', 'pytest-mock==3.1.1']
+DEV_EXTRAS = [
+    "pytest>=5.0,<=7.0.1;python_version<'3.10'",
+    "pytest==7.2.0;python_version>='3.10'",
+    "py>=1.9.0",
+    "openpyxl==2.6.4",
+    "pytest-mock==3.1.1"
```

Comment:
do we really need to install all third-party components for pytest with specific versions?

Reply:
We only added `py` (discussed in separate comment). `openpyxl` and `pytest-mock` were not changed, the lines are highlighted green because we formatted the Python list to be an entry per line. :)